repos/fdbt-site/src/interfaces/matchingJsonTypes.ts

```diff
@@ -261,6 +266,7 @@ export interface Product {
     productDurationUnits?: string;
     productEndTime?: string;
     carnetDetails?: CarnetDetails;
+    pricingByDistance?: DistancePricingData;
```

Comment:
We shouldn't need to add this here because this is saying that a Product always has a produceName and sometimes will have pricingByDistance in it. For this type of ticket, it will not have the productName on this level, it will be inside the pricingByDistance.